NICE.Identity.Management/ClientApp/src/components/UnlockUser/UnlockUser.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+import { Endpoints } from "../../data/endpoints";
+
+type UnlockUserProps = {
+	id: number;
+	isBlocked: boolean;
+	onToggleLock: Function;
```

Comment:
Can we type the onToggleLock function? Something like:

```suggestion
	onToggleLock: (user: UserType) => void;
```